music/src/core/soundfont.ts

```diff
@@ -194,7 +195,7 @@ export class Instrument {
       sampleNamesAndURLs.forEach(
           (nameAndURL) => this.buffers.add(nameAndURL.name, nameAndURL.url));
       await new Promise(resolve => Tone.Buffer.on('load', resolve));
-      console.log(`Loaded samples for ${this.name}.`);
+      logging.log(`Loaded samples for ${this.name}.`);
```

Comment:
I'd probably use SoundFont here as the prefix

Reply:
Should I use that prefix for all the logs in the file?

Reply:
yeah, i think that would be neat (it helps see where the logs are coming from)

Reply:
Ok, I added a new commit.

Should I add this to the other files as well?

Reply:
yes please!

Reply:
Ok, does my latest commit look ok?